src/vdaf/dummy.rs

```diff
@@ -331,3 +333,86 @@ impl Encode for AggregateShare {
         self.0.encoded_len()
     }
 }
+
+/// Returns the aggregate result that the dummy VDAF would compute over the provided measurements,
+/// for the provided aggregation parameter.
+pub fn expected_aggregate_result<M>(aggregation_parameter: u8, measurements: M) -> u64
+where
+    M: IntoIterator<Item = u8>,
+{
+    (measurements.into_iter().map(u64::from).sum::<u64>()) % u64::from(u8::MAX)
+        + u64::from(aggregation_parameter)
```

Comment:
```suggestion
    measurements.into_iter().map(u64::from).sum::<u64>() % (u64::from(u8::MAX) + 1)
        + u64::from(aggregation_parameter)
```

Reply:
We might also benefit from extracting `u64::from(u8::MAX) + 1` into its own constant.
```rust
const MODULUS: u64 = u8::MAX as u64 + 1;
```